lib/model/store.dart

```diff
@@ -801,6 +801,7 @@ class UpdateMachine {
             assert(debugLog('Error polling event queue for $store: $e\n'
                 'Backing off and retrying even though may be hopeless…'));
             // TODO tell user on non-transient error in polling
+            reportErrorToUserInDialog('Error loading server data. Will retry: $e');
```

Comment:
```
store: Reoprt non-transient polling errors.
```

commit-message nit: spelling

Reply:
Also this TODO can be removed:

```dart
            // TODO tell user on non-transient error in polling
```

Reply:
Also, can this be a translated string? If that's not straightforward, let's include a `TODO(i18n)`.

Reply:
Hmm. And for non-transient errors, we expect that retrying won't help, right? It seems like we should have a TODO to not retry. I'm also wondering if retrying _and_ showing a dialog, for a task that's likely to be hopeless, will spam the user with the same dialog repetitively.

Reply:
> Hmm. And for non-transient errors, we expect that retrying won't help, right? It seems like we should have a TODO to not retry. I'm also wondering if retrying and showing a dialog, for a task that's likely to be hopeless, will spam the user with the same dialog repetitively.

Yes, that's #186. Added a line mentioning that.